gridt/src/app/home/home.page.scss

```diff
@@ -7,9 +7,13 @@
     justify-content: space-between;
 }
 
+.movement-card{
+  min-width: 300px;
+}
+
 // The width of cards on the home page should match the width of the screen for small enough screens. Issue #60
-@media screen and (max-width: 600px) {
-  ion-card{
+@media screen and (max-width: 576px) {
```

Comment:
I think it would be nice to use css variables here: https://ionicframework.com/docs/theming/css-variables#using-css, it leaves more flexibility and makes the code more understandable in my opinion.

Reply:
It would indeed be perfect. But, unfortunately css variables are not yet supported for '@media' queries